src/main/java/com/sprint/mission/discodeit/mapper/api/UserApiMapper.java

```diff
@@ -24,11 +24,13 @@ public FindUserResponse toFindUserResponse(UserDTO.User user) {
                 .contentType(user.getProfileId().getContentType())
                 .build() :
             null)
+        .role(user.getRole())
         .isOnline(user.getIsOnline())
```

Comment:
회원가입 시 처음엔 `user.getIsOnline()`이 null이라서 아래처럼 바꿔줘야 회원가입 로직이 정상동작하더라구요.
```suggestion
        .isOnline(BooleanUtils.isTrue(user.getIsOnline()))
```

Reply:
아래 요구사항을 위해 로그인 여부를 세션에서 가져오면 좋을 것 같아요!
```
[ ] UserStatus 엔티티 대신 SessionRegistry를 활용해 사용자의 로그인 여부를 판단하도록 리팩토링하세요.
```